pkg/kubecost/allocation.go

```diff
@@ -2405,3 +2405,67 @@ func (asr *AllocationSetRange) Window() Window {
 
 	return NewWindow(&start, &end)
 }
+
+// Start returns the earliest start of all Allocations in the AllocationSetRange.
+// It returns an error if there are no allocations.
+func (asr *AllocationSetRange) Start() (time.Time, error) {
+	start := time.Date(1970, 1, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
Why this time and not the zero value for `time.Time`?

Reply:
No reason in particular. Switched to zero value.